app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import json
+
+from decimal import Decimal as Dec
+
+
+def calculate_profit(name_of_file):
+    with open(name_of_file, "r") as f:
+        trades_info = json.load(f)
+
+    profit_info = {
+        "earned_money": Dec("0"),
+        "matecoin_account": Dec("0"),
+    }
+
+    for trade in trades_info:
+        if trade["bought"]:
+            profit_info["matecoin_account"] += Dec(trade["bought"])
+            profit_info["earned_money"] -= (
+                Dec(trade["bought"]) * Dec(trade["matecoin_price"])
```

Comment:
you may create variable for Dec(trade["matecoin_price"]) and use it in both if statements